diff --git a/src/lib/elements/forms/inputDigits.svelte b/src/lib/elements/forms/inputDigits.svelte
index dee662d63..7d54930f5 100644
--- a/src/lib/elements/forms/inputDigits.svelte
+++ b/src/lib/elements/forms/inputDigits.svelte
@@ -25,9 +25,15 @@
     });
 
     onMount(() => {
-        if (element && autofocus) {
-            element.querySelector('input')?.focus();
-        }
+        const interval = setInterval(() => {
+            const input = element.querySelector('input');
+            if (element) {
+                if (input && autofocus) {
+                    input.focus();
+                }
+                clearInterval(interval);
+            }
+        }, 10);
     });
 </script>
 
diff --git a/src/routes/+layout.svelte b/src/routes/+layout.svelte
index 99cc475ff..c8c0239c7 100644
--- a/src/routes/+layout.svelte
+++ b/src/routes/+layout.svelte
@@ -54,6 +54,7 @@
                 '/recover',
                 '/invite',
                 '/card',
+                '/mfa',
                 '/hackathon'
             ];
             if ($user) {
